src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -88,9 +87,23 @@
     <Compile Include="$(CommonPath)\Interop\Windows\Crypt32\Interop.Alg.cs" >
       <Link>Common\Interop\Windows\Crypt32\Interop.Alg.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs" >
-      <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\Interop.SchProtocols.cs" >
+      <Link>Common\Interop\Windows\SChannel\Interop.SchProtocols.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\Winsock\Interop.ErrorCodes.cs" >
```

Comment:
As per the interop guidelines, constants related to particular libraries should live with the other interop code for those libraries. Constants from this file are used in SocketError.cs.